bob-common/src/node.rs

```diff
@@ -93,6 +93,10 @@ impl Node {
         }
     }
 
+    pub fn connection_is_set(&self) -> bool {
```

Comment:
Should be replaced with `connection_available` when PR https://github.com/qoollo/bob/pull/630 will be merged